main_app/services.py

```diff
@@ -1,9 +1,15 @@
 from django.http import JsonResponse
 import requests
 
-def get_parks(self, pk):
-    info = requests.get('https://jsonplaceholder.typicode.com/todos/1').json()
+def get_parks(self, coordinates='', query=''):
+
+    radius = 2000
+    endpoint_url = "https://maps.googleapis.com/maps/api/place/nearbysearch/json?fields=photos,formatted_address,name,rating,opening_hours,geometry&key=AIzaSyAb6K8lFSR2jjUv5GwqKvTMgdumdWPh-FE&keyword=%s&location=%s&radius=2000" % (query,coordinates)
+
```

Comment:
@nicolegooden `Keyword` is currently set to 'query', but can be set to `dog+parks`